.drone.yml

```diff
@@ -187,43 +226,92 @@ trigger:
     - develop
 
 ---
-##############################################################################################################
 kind: pipeline
-name: unit-tests
+name: test-protocol-switch-1-to-2
```

Comment:
@adonagy 
just a cosmetics, I would name it like: `mainnet-protocol-switch-1-to-2-test` :)